hawkular-alerts-rest-tests/src/test/groovy/org/hawkular/alerts/rest/EventsITest.groovy

```diff
@@ -58,6 +58,9 @@ class EventsITest extends AbstractITestBase {
 
         resp = client.get(path: "events", query: [tags:"tag-01|value-01,tag-02|value-02",thin:true] )
         assertEquals(200, resp.status)
+
+        resp = client.get(path: "events", query: [tagQuery:"tagA or (tagB and tagC in ['e.*', 'f.*'])"] )
```

Comment:
A naive question but I guess when you see something like "tagA" it just means the tag with name "tagA" and any value?  So in a lot of the tests here we're just doing matching against a name and any value?  Am I right that the test above logically expands to:

name=tagA:value=* or ( name=tagB:value=* and ( name=tagC:value=e.* or name=tagC:value=f.* ))